integration_tests/savings_kpi.feature

```diff
@@ -8,5 +8,5 @@ Feature: Savings KPI integration tests
   Examples: Cases
     | scenario                          | setup                               | saving_percentage |
     | Young Couple House                | kpi.d3asim_3464_young_couple        |     16.6667      |
-    | Family 2 children with PV         | kpi.d3asim_3464_family_with_pv      |     43.537       |
+    | Family 2 children with PV         | kpi.d3asim_3464_family_with_pv      |     28.427       |
```

Comment:
I do not have a problem per se by this, however I am curious as to why it happened. As far as I understand, the current MIN_OFFER_AGE value for this test is still 0, therefore it should be able to broadcast immediately every update in the energy rate, therefore the deleting + reposting of the PV offer on every tick (the behavior that the commit solved) should in theory have exactly the same effect as reposting the event only during update_intervals. I would recommend to leave it as is, and open a ticket in order to further debug this behavior.